src/snowcli/utils.py

```diff
@@ -25,6 +27,9 @@
 from jinja2 import Environment, FileSystemLoader
 from snowflake.connector.cursor import SnowflakeCursor
 
+if sys.platform == "win32":
+    from ctypes import create_unicode_buffer, windll  # type: ignore
```

Comment:
Maybe move import to if in path_resolver?

Reply:
Moved